args/physics/colliders/convexcollider.cpp

```diff
@@ -1,49 +1,96 @@
 #include <physics/colliders/convexcollider.hpp>
 #include <physics/PhysicsStatics.h>
 
+#include <physics/data/convexconvexpenetrationquery.h>
+#include <physics/data/edgepenetrationquery.h>
+
 namespace args::physics
 {
     void ConvexCollider::CheckCollisionWith(ConvexCollider* convexCollider, physics_manifold& manifold) 
     {
-      
 
+        //--------------------- Check for a collision by going through the edges and faces of both polyhedrons  --------------//
         //'this' is colliderA and 'convexCollider' is colliderB
 
-        HalfEdgeFace* ARefFace = nullptr;
+        HalfEdgeFace** ARefFace = &convexCollider->GetHalfEdgeFaces().at(0);
```

Comment:
is this actually supposed to be pointing to the first item in the container, or just to the data that's in the container? otherwise maybe use a .data() function or something like that.

Reply:
it doesnt actually matter which item it initially points to. I just needed to initialize it with something because in the function below it its treated as an out parameter.